modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/exp/RexImpTable.java

```diff
@@ -574,6 +585,13 @@ private void defineMethod(SqlOperator operator, Method method, NullPolicy nullPo
         map.put(operator, new MethodImplementor(method, nullPolicy, false));
     }
 
+    /** */
+    private ReflectiveImplementor defineReflective(SqlOperator operator, Method... methods) {
```

Comment:
where do we call `constructorSupplier(Class<T> klass)` ?

Reply:
It was required for aggregates in calcite, but we handle aggregates in a special way. Removed.